src/options.html

```diff
@@ -144,120 +144,120 @@
-                of the extension for more details on how this works.
-                <i>Note: Currently, externally hosted pull request templates
-                    outside of Bitbucket's domain is not supported with the
-                    Firefox addon.</i>
-                <input type="text" name="prTemplateUrl" style="width: 100%;" />
-            </label>
-            <br />
-            <br />
+        <label>
+            <input type="checkbox" name="prTemplateEnabled" /> Use a
+            <em>PULL_REQUEST_TEMPLATE.md</em> from the repository when
+            creating new pull requests or specify a URL with your own raw
+            template (must be hosted publicly in
+            <a href="http://gist.github.com/">http://gist.github.com/</a>).
+            See the
+            <a href="http://github.com/refined-bitbucket/refined-bitbucket">README</a>
+            of the extension for more details on how this works.
+            <i>Note: Currently, externally hosted pull request templates
+                outside of Bitbucket's domain is not supported with the
+                Firefox addon.</i>
+            <input type="text" name="prTemplateUrl" style="width: 100%;" />
+        </label>
+        <br />
+        <br />
 
-            <label>
-                <input type="checkbox" name="mergeCommitMessageEnabled" /> Use a
-                <em>MERGE_COMMIT_TEMPLATE</em> from the repository or specify a
-                URL with your own raw template where you can use variables like
-                `id`, `title`, `description` and more. See the
-                <a href="http://github.com/refined-bitbucket/refined-bitbucket">README</a>
-                of the extension for more details on how this works.
-                <i>
-                    Note: Currently, externally hosted templates (outside from
-                    Bitbucket's domain) are only tested with
-                    <a href="http://gist.github.com/">http://gist.github.com/</a>, and are not supported with the
-                    Firefox addon.
-                </i>
-                <input type="text" name="mergeCommitMessageUrl" style="width: 100%;" />
-            </label>
-            <br />
-            <br />
+        <label>
+            <input type="checkbox" name="mergeCommitMessageEnabled" /> Use a
+            <em>MERGE_COMMIT_TEMPLATE</em> from the repository or specify a
+            URL with your own raw template where you can use variables like
+            `id`, `title`, `description` and more. See the
+            <a href="http://github.com/refined-bitbucket/refined-bitbucket">README</a>
+            of the extension for more details on how this works.
+            <i>
+                Note: Currently, externally hosted templates (outside from
+                Bitbucket's domain) are only tested with
+                <a href="http://gist.github.com/">http://gist.github.com/</a>, and are not supported with the Firefox
```

Comment:
my linter keeps playing with this file on save but look like it did the right indentation for this time

Reply:
Added the `.html` extension to the files to be prettified. Maybe that'll resolve the issue moving forward.